ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -145,10 +145,8 @@ public static BeaconStateWithCache initialize_beacon_state_from_eth1(
 
   public static BeaconStateWithCache initialize_beacon_state_from_eth1_new(
       Bytes32 eth1_block_hash, UnsignedLong eth1_timestamp, List<? extends Deposit> deposits) {
-    UnsignedLong genesis_time =
-        eth1_timestamp
-            .minus(eth1_timestamp.mod(UnsignedLong.valueOf(SECONDS_PER_DAY)))
-            .plus(UnsignedLong.valueOf(2).times(UnsignedLong.valueOf(SECONDS_PER_DAY)));
+    UnsignedLong genesis_time = eth1_timestamp;
```

Comment:
Why did you change this? It strays from the spec: https://github.com/ethereum/eth2.0-specs/blob/v0.8.3/specs/core/0_beacon-chain.md#genesis

Reply:
good catch...i will need to look that this again.  It is potentially fallout from the confusion for the different startup modes we have.  Before that change, I was having issues with the way the slot was calculated when trying to do a sync'd start from genesis.  I saw how it was worked around before (i think it is just overwritten in another part of the code), so I might need to use a similar tactic